web/README.md

```diff
@@ -40,3 +40,9 @@ npm install
 make dev
 make server
 ```
+
+The workflow for interactive development of just one app in debug mode:
+
+```
+./bin/build-wasm game abstreet && rm -rf build/dist/abstreet/wasm_pkg/ && cp -Rv src/abstreet/wasm_pkg build/dist/abstreet/ && make server
```

Comment:
You should also be able to do:

```
make abstreet server

# or if you want a clean build
make clean abstreet build/dist/data server

# maybe the `build/dist/data` symlink should be a pre-req of the `server` rule?
```

Reply:
That second command did the trick! For some reason, I was expecting `make clean` to totally nuke the wasm target directory and cause a complete rebuild from scratch. But that's not what the makefile rule does. This works, thank you!